src/App.css

```diff
@@ -1,3 +1,10 @@
+#root {
+  max-width: 550px;
+  height: 100vh;
+  margin: 0 auto;
+  font-size: 62.5%;
+}
+
 .App {
   text-align: center;
 }
```

Comment:
I think it would be better to inject these with [`createGlobalStyles`](https://www.styled-components.com/docs/api#createglobalstyle) component of styled-components

[Here's an example](https://github.com/RodPaDev/rodpa-dev/blob/master/src/GlobalStyle.js)
[and here's how to implement it into index.js](https://github.com/RodPaDev/rodpa-dev/blob/master/src/index.js)